cms/envs/common.py

```diff
@@ -761,6 +764,9 @@
     'openedx.core.djangoapps.credit',
 
     'xblock_django',
+
+    # edX Proctoring
+    'edx_proctoring',
```

Comment:
I've been setting up these IDA-type apps in the 'optional apps' area -- not sure if you want to follow that pattern or not